### PR TITLE
S14wc

### PR DESCRIPTION
**Pull request by bcli4d**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -71,6 +71,6 @@ ADD . /app
 RUN pip3 install -r /app/requirements.txt -t /app/lib/ --upgrade
 RUN pip3 install gunicorn==19.9.0
 
-ENV PYTHONPATH=/app:/app/apiv4:/app/lib:/app/IDC-Common:${PYTHONPATH}
+ENV PYTHONPATH=/app:/app/apiv1:/app/lib:/app/IDC-Common:${PYTHONPATH}
```
Comment (s-paquette) [inline Dockerfile:75]: This won't work; the folder is api not apiv1. So /app/api: and then below, api:app (unless you've changed it to apiv1 but it doesn't appear to be the case).

**Review by s-paquette — COMMENTED** [review]
Note, it's also saying this is out of date with master and needs to do a pull.

**Review by s-paquette — CHANGES_REQUESTED** [review]
Need to fix the apiv1 bit.

**Comment by bcli4d** [comment]
Done.

**Review by s-paquette — APPROVED** [review]

**Pull request merged**